src/buttons/ButtonGroup.js

```diff
@@ -37,7 +39,10 @@ const ButtonGroup = props => {
     ...attributes
   } = props;
 
-  const Component = component || TouchableHighlight;
+  const Component =
+    component || Platform.OS === 'ios'
```

Comment:
Lets move this logic to the `defaultProps` section at the bottom

Reply:
sure !!!